htsget_server/database.py

```diff
@@ -486,7 +486,7 @@ def create_drs_object(obj, tries=1):
             session.add(new_object)
             session.commit()
 
-            # if we have reference_genome info, it's a GenomicDrsObject and needs a variantfile:
+            # if we have reference_genome info, it's a AnalysisDrsObject and needs a variantfile:
```

Comment:
we can do this, even though there Analyses with reference_genomes that produce outputs other than variantfiles because we get here only when dealing with VCFs?